app/assets/javascripts/discourse/components/topic-list-item.js.es6

```diff
@@ -35,6 +35,34 @@ export const ListItemDefaults = {
   attributeBindings: ["data-topic-id"],
   "data-topic-id": Ember.computed.alias("topic.id"),
 
+  init() {
+    this._super(...arguments);
+
+    if (typeof this.get("topic.read_by_group_member") !== "undefined") {
```

Comment:
Is there any reason

```
if (this.get("topic.read_by_group_member")) {}
```

wouldn't work ?

Reply:
We want to be subscribed even if  `topic.read_by_group_member` is false, that's why we explicitly check for undefined.